stacks-signer/src/runloop.rs

```diff
@@ -899,12 +992,14 @@ impl<C: Coordinator> SignerRunLoop<Vec<OperationResult>, RunLoopCommand> for Run
         if let Some(command) = cmd {
             self.commands.push_back(command);
         }
-        // TODO: This should be called every time as DKG can change at any time...but until we have the node
-        // set up to receive cast votes...just do on initialization.
+
+        // TODO: need to update dkg more frequently than just on initialization
```

Comment:
Yes -- you should be checking this every Bitcoin block or so

Reply:
I might be checking it a bit too excessively right now. but i do on every pass of the runloop essentially.